Source/Scene/GroundPrimitive.js

```diff
@@ -807,6 +903,19 @@ define([
         }
     }
 
+    GroundPrimitive._initialized = false;
+    GroundPrimitive.init = function() {
```

Comment:
Obviously, a more radical approach would be to have a general "initialize" function for all of Cesium we have users call before they do anything, but that's probably a little too extreme.